index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio-Ishpaul Singh</title>
+</head>
+<body>
+    <header>
+        <!--The nav bar-->
+        <nav>
+            <!--logo-->
+            <div class="logo">
+                <h1>ISHPAUL SINGH</h1>
+            </div>
+            <!--menu icon-->
+            <div class="toggle-menu">
+                <a href="#"><img src="assets/icons/menu-icon.svg" alt="menu"></a>
+            </div>
+        </nav>
+
+    </header>
+        <!--INTRODUCTION SECTION-->
+        <section class="intro-section">
+            <!--HEADLINE-->
+            <h1 class="headline">
+                Hii!<br>
+                I'm Ishpaul Singh<br>
+                Glad to see you!
+            </h1>
```

Comment:
- [ ] You should have only one `h1`  tag in your HTML codes. This will improve the semantics of your project.